.ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -97,7 +97,11 @@ export GOOGLE_TEST_DIRECTORY=$(go list ./... | grep -v github.com/hashicorp/terr
 echo "checking terraform version"
 terraform version
 
-
+go build $GOOGLE_TEST_DIRECTORY
+if [ $? != 0 ]; then
+  echo "Skipping tests: Build failure detected"
```

Comment:
VCR status will remain pending at this point due to line 55. Could we post the pending status after we run confirm build-ability?